cypress/plugins/index.js

```diff
@@ -16,7 +16,7 @@
  * @type {Cypress.PluginConfig}
  */
 const { lighthouse, prepareAudit } = require('cypress-audit')
-const dotenvPlugin = require('cypress-dotenv')
```

Comment:
Possibly misunderstanding here but why is cypress config being changed in order to add Sentry? Or this a bonus extra

Reply:
You have answered my question below

Reply:
The `cypress-dotenv` plugin only supports the one .env file but we need to differentiate between dev, test and production so that the sentry webpack plugin only uploads the source maps when a release is built on CI